packages/ERTP/src/amountMath.js

```diff
@@ -164,25 +179,52 @@ const checkLRAndGetHelpers = (leftAmount, rightAmount, brand = undefined) => {
 };
 
 /**
+ * @template {AmountValue} V
  * @param {MathHelpers<AmountValue>} h
- * @param {Amount} leftAmount
- * @param {Amount} rightAmount
- * @returns {[AmountValue, AmountValue]}
+ * @param {Amount<V>} leftAmount
+ * @param {Amount<V>} rightAmount
+ * @returns {[V, V]}
  */
 const coerceLR = (h, leftAmount, rightAmount) => {
+  // @ts-ignore cast (ignore b/c erroring in CI but not my IDE)
   return [h.doCoerce(leftAmount.value), h.doCoerce(rightAmount.value)];
 };
 
-/** @type {AmountMath} */
+/**
+ * Logic for manipulating amounts.
+ *
+ * Amounts are the canonical description of tradable goods. They are manipulated
+ * by issuers and mints, and represent the goods and currency carried by purses
+ * and
+ * payments. They can be used to represent things like currency, stock, and the
+ * abstract right to participate in a particular exchange.
+ */
 const AmountMath = {
+  /**
+   * Make an amount from a value by adding the brand.
+   *
+   * @template {AmountValue} [V=AmountValue]
+   * @param {Brand} brand
+   * @param {V extends NatValue ? NatValue : V extends SetValue ? SetValue : V extends CopySetValue ? CopySetValue : V extends CopyBagValue ? CopyBagValue : never} allegedValue
```

Comment:
What is the benefit of this over `@param {V} allegedValue` ?

The cost is plain: it's long and hard to read.

Reply:
For `allegedX` parameters, the function body is best served by a type of `unknown`. But that doesn't help callers as much.

Reply:
I tried to explain the comment on line 211. Without this condition, a literal like `1n` for `allegedValue` will bind `V` such that the return of make() would be `Amount<1n>`. That's okay in lots of cases but it breaks the math functions.